### DISCUSSION TITLE
Availability of Framewise Displacement from the vNavs

### DISCUSSION THREAD

**Discussion by xiyan-stat** [now-i-know-my-abcd/docs#95] [Stats & Code Help]
Hi everyone, I'm working with ABCD data and would like to clarify whether the framewise displacement (FD) values derived from the vNavs (volumetric navigators) are available for the Siemens scanner sites. And if so, I'm interested in knowing where I can find vNavs-based FD or other motion related metrics from vNavs in the data files? Thank you so much!

**Comment by laikevin128** [comment]
Hi, thank you for your post- we do not have FD values extracted from Siemens vNavs.

**Reply by xiyan-stat** [reply]
Thank you so much for your answer! Are the vNavs available in the raw imaging data so that we can calculate them?
